flaskapp/blueprints/errors/routes.py

```diff
@@ -2,17 +2,17 @@
 
 errors = Blueprint('errors', __name__)
 
-
+#Display error 404
```

Comment:
@errors.app_errorhandler(404)
def error_404(error):
    """
    return render_template('errors/404.html'), 404

Reply:
It will activate auto docstring and type bunch of detail for you fill that details